test/verify/check-kdump

```diff
@@ -175,6 +175,62 @@ class TestKdump(MachineCase):
         self.assertNotEqual(
             m.execute("""find "{0}" -maxdepth 1 -mindepth 1 -type d -exec echo {{}} \;""".format(customPath)), "")
 
+    @nondestructive
+    def testConfiguration(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_file("/etc/kdump.conf")
+
+        m.execute("systemctl disable --now kdump || true")
+
+        self.login_and_go("/kdump")
+        b.wait_visible("#app")
+
+        # Check remote ssh location
+        b.click("#kdump-change-target")
+        b.wait_visible("#kdump-settings-dialog")
+        b.set_val("#kdump-settings-location", "ssh")
+        b.set_input_text("#kdump-settings-ssh-server", "admin@localhost")
+        b.set_input_text("#kdump-settings-ssh-key", "/home/admin/.ssh/id_rsa")
+        b.set_input_text("#kdump-settings-local-directory", "/var/tmp/crash")
+        b.click("button:contains('Apply')")
+        b.wait_not_present("#kdump-settings-dialog")
+        wait(lambda: "path /var/tmp/crash" in m.execute("cat /etc/kdump.conf"))
```

Comment:
Seems unusual that this file isn't written when the dialog goes down, but that may be a bug in our implementation. But do we really need to cat the file 3 times, and even 4 times below? It should only write the file once, so after the first condition is true, the others ought to as well?

Reply:
Good point. It indeed is synchronized by the dialog disappearing, so no need for waiting there. Removed it.